src/app/core/view/fa-dynamic-icon/fa-dynamic-icon.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, Input } from "@angular/core";
+import { IconDefinition, IconName } from "@fortawesome/fontawesome-svg-core";
+import {
+  faChartLine,
+  faChild,
+  faHome,
+  faQuestionCircle,
+  faTable,
+  faUniversity,
+  faWrench,
+} from "@fortawesome/free-solid-svg-icons";
+import {
+  faCalendar,
+  faCalendarCheck,
+  faFileAlt,
+  faUser,
+} from "@fortawesome/free-regular-svg-icons";
+import { FaIconLibrary } from "@fortawesome/angular-fontawesome";
+
+/**
+ * font-awesome icons decoupled from the config
+ */
+const faDynamicIcons = new Map<string, IconDefinition>([
+  ["home", faHome],
+  ["child", faChild],
+  ["university", faUniversity],
+  ["calendar", faCalendar],
+  ["calendar-check", faCalendarCheck],
+  ["table", faTable],
+  ["notes", faFileAlt],
+  ["wrench", faWrench],
+  ["user", faUser],
+  ["question", faQuestionCircle],
+  ["line-chart", faChartLine],
+]);
+
+/**
+ * This component can be used to display dynamic Font-Awesome icons.
+ * The term 'dynamic icons' refers to icons that are injected at runtime,
+ * for example through the config.
+ */
+@Component({
+  selector: "app-fa-dynamic-icon",
+  template: `<fa-icon [icon]="_icon"></fa-icon>`,
+})
+export class FaDynamicIconComponent {
+  /** The fallback icon if the given icon is neither known (inside the internal map)
+   * nor registered as a font-awesome icon
+   */
+  static fallbackIcon = faQuestionCircle;
+
+  /**
+   * Sets the dynamic icon by name.
+   * You should make sure that the icon is registered inside the {@link faDynamicIcons}-map,
+   * or put it into this map if it isn't there.
```

Comment:
What is the benefit of always registering the icon instead of 'just' using it with the normal name?

Reply:
To have it decoupled from the implementation. 
When it comes to change icon definitions, it is a trade-of between a regular update or a config-migration that needs to be executed. I have chosen the former one.

Reply:
This adds quite a bit of complexity. If it requires code changes when adding some non-standard custom icon for an individual customer, I think this would be a barrier rather than a help for our current state of operations